README.md

```diff
@@ -174,8 +177,22 @@ Indexes several days of logs with a fixed (raw) logging volume per day and runni
 | `bulk_size`             | Number of documents to send per bulk                                                                                                   | `int` | `1000`                |
 | `daily_logging_volume`  | The raw logging volume. Supported units are bytes (without any unit), `kb`, `MB` and `GB`). For the value, only integers are allowed.  | `str` | `100GB`               |
 | `starting_point`        | The first timestamp for which logs should be generated.                                                                                | `str` | `2018-05-25 00:00:00` |
-| `number_of_days`        | The number of simulated days for which data should be generated.                                                                       | `int` | `24`                  |
-| `number_of_shards`      | Number of primary shards                                                                                                               | `int` | `3`                   |
+| `number_of_days`        | The number of simulated days for which data should be generated.                                                                       | `int` | `6`                  |
+
+### index-fixed-load-and-query
+
+Indexes (several days of) logs at a fixed target throughput using a fixed (raw) logging volume per day while running queries concurrently.  Requires executing `index-logs-fixed-daily-volume` first. This challenge will end as soon the indexing task has completed. The table below shows the track parameters that can be adjusted along with default values:
+
+| Parameter                    | Explanation                                                                                                                            | Type  | Default Value         |
+| ---------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | ----- | --------------------- |
+| `bulk_indexing_reqs_per_sec` | Number of bulk indexing requests/sec. Multiply this by bulk_size to understand indexing throughput in docs/s.                          | `int` | `20`                  |                                                    
+| `bulk_size`                  | Number of documents to send per bulk                                                                                                   | `int` | `1000`                |
+| `bulk_indexing_clients`      | Number of bulk indexing clients/connections                                                                                            | `int` | `8`                   |
+| `search_clients`             | Number of search clients/connections                                                                                                   | `int` | `1`                   |
```

Comment:
Maybe we should clarify that each of the three queries will be run with that many clients? I.e. with `search_clients` set to `2`, we will actually run with six clients because there are three searches and each of them is run with two clients.